123_state_tasks.R

```diff
@@ -83,3 +93,20 @@ split_inventory <- function(
     state_files = write_individual_tmp_files(sites_info)
     scipiper::sc_indicate(ind_file = summary_file, data_file = state_files)
 }
+
+combine_obs_tallies <- function(...){
+    # filter to just those arguments that are tibbles (because the only step
+    # outputs that are tibbles are the tallies)
+    dots <- list(...)
+    tally_dots <- dots[purrr::map_lgl(dots, is_tibble)]
+    combined = bind_rows(tally_dots)
+    return(combined)
+}
+
+summarize_timeseries_plots <- function(ind_file, ...){
+    # filter to just those arguments that are character strings (because the only
+    # step outputs that are characters are the plot filenames)
+    dots <- list(...)
+    plot_dots <- dots[purrr::map_lgl(dots, is.character)]
+    do.call(combine_to_ind, c(list(ind_file), plot_dots))
```

Comment:
The version of this custom function that would create an object rather than a file would be:
```r
summarize_timeseries_plots <- function(...){
     # filter to just those arguments that are character strings (because the only
     # step outputs that are characters are the plot filenames)
     dots <- list(...)
     plot_dots <- dots[purrr::map_lgl(dots, is.character)]
     do.call(combine_to_ind, c(list(''), plot_dots))
}
```